src/main/java/de/szut/lf8_project/lf8/project/ProjectService.java

```diff
@@ -0,0 +1,39 @@
+package de.szut.lf8_project.lf8.project;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class ProjectService {
+    private final ProjectRepository repository;
+
+    public ProjectService(ProjectRepository repository) {
+        this.repository = repository;
+    }
+
+    public ProjectEntity create(ProjectEntity entity) {
+        return repository.save(entity);
+    }
+
+    public List<ProjectEntity> readAll() {
+        return this.repository.findAll();
+    }
+
+    public ProjectEntity readById(long id) {
+        Optional<ProjectEntity> optionalQualification = this.repository.findById(id);
+        if (optionalQualification.isEmpty()) {
+            return null;
+        }
+        return optionalQualification.get();
```

Comment:
Wenn es hier um optionale Qualificationen geht, warum wird dann ein Projekt abgerufen?

Reply:
Brudi, dass fixt nicht das angesprochene Problem. Warum machst du hier was mit optionalQualification wenn es um die Abfrage von Projekten geht qwq

Reply:
Und das wir da null returnen ist schon in ordnung finde ich. Wenn wir einfach ne Exception werfen ist es inkonsistent, weil wir an manchen stellen null returnen und an anderen ne exception thrown

Reply:
Wait i just realized the variable is named optionalQualification. hold on. ¬_¬